webview/package.json

```diff
@@ -13,7 +13,7 @@
     "test": "jest --collect-coverage",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build --webpack-stats-json",
-    "svgr": "svgr --out-dir src/shared/components/icons --ignore-existing ../node_modules/@vscode/codicons/src/icons && svgr -d src/shared/components/icons icons/"
```

Comment:
This is my first time actually seeing this command! Looking at it, I'm assuming we used this to generate chosen icon components? I've just been creating them manually 🤣

Reply:
Yes, that is why I also added steps in the contributing guide. To be honest, I've been creating them manually as well for a while to because the process was broken. Running the command would generate every codicon, you would then have to remove all the ones you don't need.